content/hackathons/usw24/index.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Unikraft Summer Workshop 2024"
+description: Unikraft Summer Workshop 2024, July 1 - July 20, 2024
+startDate: 2024-07-01
+registerForm: https://forms.gle/6AMFoCoqJqKGdBgX6
```

Comment:
```suggestion
registerForm: https://forms.gle/6AMFoCoqJqKGdBgX6
```

Should this be `registrationForm`?

Reply:
The registration form is not available.

They should submit their GitHub ID, affiliation, e-mail address, (equivalent) year of study and motivation for taking part in the hackathon.

Reply:
Fixed, it was already created, but I forgot to enable it.

Reply:
> Should this be registrationForm?

No, this is how the renderer likes it.